docs/end-user/filtering-call-trees.md

```diff
@@ -0,0 +1,81 @@
+# Filtering call trees
+
+Call trees can grow to be quite large, especially when profiling a browser engine. Typically only portions of the tree are actually useful for any given analysis. perf.html provides a variety of tools for filtering and transforming a tree in order to only show the relevant parts. It's important when filtering to know the difference between filtering out samples and transforming stacks. When removing samples, the tree will essentially be the same shape, but may be missing certain branches that were only present in certain samples. When transforming the tree, the shape of the sample's stacks will be modified. Samples will only be dropped if their stacks are completely removed during the transformation operation.
+
+The following are the different types of filtering operations that are supported.
+
+| Filter type | Description |
+| ----------- | ----------- |
+| Search filter | Drop samples that do not match a text string. |
+| Implementation filter | Restrict stacks to an implementation—native (C++) stacks, or to JavaScript stacks. |
```

Comment:
This looks like en em dash in "implementtion-native" while we want a normal dash here.

Reply:
I believe `em` dash is the appropriate one here: http://www.thepunctuationguide.com/em-dash.html

> A pair of em dashes can replace a pair of parentheses. Dashes are considered less formal than parentheses; they are also more intrusive. If you want to draw attention to the parenthetical content, use dashes. If you want to include the parenthetical content more subtly, use parentheses.